spec/presenters/nsm/check_answers/case_details_card_spec.rb

```diff
@@ -15,15 +15,15 @@
 
   describe '#row_data' do
     it 'generates case detail rows with no remittal' do
-      expect(subject.row_data).to eq(
+      expect(subject.row_data).to match(
         [
           {
             head_key: 'main_offence',
             text: 'Theft'
           },
           {
             head_key: 'main_offence_date',
-            text: '1 January 2023'
+            text: /\A\d{1,2} \w+ \d{4}\z/
```

Comment:
This feels to me like it weakens the test. "0 Banana 0000" would pass. Since we're here testing the precise formatting of dates (and tests like this allowed us to spot where there was a formatting error with leading zeroes being present in the day numbers), I think it's reasonable that these tests would be premised on having an explicit date provided in the test setup, and a matching string representation of that specific date being expected in the test details.

Reply:
yeah, it was find with `yyyy-mm-dd` but these felt more flaky... I will set a explicit date for these tests.